### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by ErikStoupignan**
# Add views 

- Use preprocessed HTML file with embedded Ruby code. ✅ 
- Use layouts and templates for shared content. ✅ 

### http://localhost:3000/
image

### http://localhost:3000/users/1
image

### http://localhost:3000/users/1/posts
image

### http://localhost:3000/users/1/posts/1
image

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+      <h2 class="post_counter">Number of posts: <%= @user.posts_counter %></h2>
+    </div>
+  </section>
+
+  <section class="border general_container margin_10px padding_10px">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </section>
+
+  <% @user.posts.each_with_index do |post, index| %>
```
Comment (omar-muhamad) [inline app/views/users/show.html.erb]: - [x] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_post` method you created on the user model to render the 3 most recent posts only.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+        <h3>Post: #<%= index + 1 %></h3>
+        <p><%= post.text %></p>
+        <div class=" display_flex">
+          <p>Likes: <%= post.likes_counter %> Comments: <%= post.comments_counter %></p>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+
+  <button class="button_post margin_10px">See all posts</button>
```
Comment (omar-muhamad) [inline app/views/users/show.html.erb]: - [x] Kindly make the `See all posts` button open the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).
[Hint] You can use the [link_to](https://www.rubyguides.com/2019/05/rails-link_to-method/) method to do it.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔
- You added seeds in your app. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/users/show.html.erb (hunk 2 above)
Comment (ErikStoupignan) [reply app/views/users/show.html.erb]: @Omar-Muhamad 
Hello! Thank you very much for your recommendations, I am finishing applying them.

File: app/views/users/show.html.erb (hunk 1 above)
Comment (ErikStoupignan) [reply app/views/users/show.html.erb]: @Omar-Muhamad 
Thank you very much for your recommendations! I just applied the suggested changes.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @ErikStoupignan, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**